.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/*
```

Always nice to have an empty line at the end of the file
```suggestion
logs/*

```